aten/src/ATen/native/quantized/cpu/qconv.cpp

```diff
@@ -169,485 +699,54 @@ class QConvInt8 final {
-              output_shape[0],
-              output_shape[1],
-              output_shape[2],
-              output_shape[3],
-              output_shape[4],
-              device(kCPU).dtype(kQUInt8),
-              output_scale,
-              output_zero_point);
-    Tensor buffer = at::empty(output.sizes(), output.options().dtype(at::kInt));
-    const int num_tasks = at::get_num_threads();
-    at::parallel_for(0, num_tasks, 1, [&](int64_t begin, int64_t end) {
-      fbgemm::DoNothing<> kNoOpObj{};
-      for (int task_id = begin; task_id < end; ++task_id) {
-        if (pack_data.q_scheme == kPerTensorAffine) {
-          fbgemm::ReQuantizeOutput<
-              kReluFused,
-              fbgemm::QuantizationGranularity::TENSOR,
-              float>
-              output_proc_obj(
-                  kNoOpObj,
-                  output_multiplier_float.data(),
-                  output_zero_point,
-                  act_zero_point,
-                  pack_data.w_zp.data(),
-                  nullptr, /* row offset buffer */
-                  col_offsets.data(),
-                  bias_data,
-                  M,
-                  groups,
-                  act_times_w_scale.data());
-          fbgemm::fbgemmConv<decltype(output_proc_obj), kSpatialDim, int32_t>(
-              conv_p,
-              act_data,
-              *pack_w,
-              reinterpret_cast<uint8_t*>(output.data_ptr<c10::quint8>()),
-              buffer.data_ptr<int32_t>(),
-              output_proc_obj,
-              task_id /* thread_id*/,
-              num_tasks /* num_threads */);
-        } else if (pack_data.q_scheme == kPerChannelAffine) {
-          fbgemm::ReQuantizeOutput<
-              kReluFused,
-              fbgemm::QuantizationGranularity::OUT_CHANNEL,
-              float>
-              output_proc_obj(
-                  kNoOpObj,
-                  output_multiplier_float.data(),
-                  output_zero_point,
-                  act_zero_point,
-                  pack_data.w_zp.data(),
-                  nullptr, /* row offset buffer */
-                  col_offsets.data(),
-                  bias_data,
-                  M,
-                  groups,
-                  act_times_w_scale.data());
-
-          fbgemm::fbgemmConv<decltype(output_proc_obj), kSpatialDim, int32_t>(
-              conv_p,
-              act_data,
-              *pack_w,
-              reinterpret_cast<uint8_t*>(output.data_ptr<c10::quint8>()),
-              buffer.data_ptr<int32_t>(),
-              output_proc_obj,
-              task_id /* thread_id*/,
-              num_tasks /* num_threads */);
-        }
-      }
-    });
-
-    return output;
+    TORCH_CHECK(false, "Arguments [stride, padding, dilation, groups] in \
+     ops.quantized.conv" + c10::to_string(kSpatialDim) + "d, " +
+     "ops.quantized.conv" + c10::to_string(kSpatialDim) + "d_relu, \
+     have been removed, please update your model to remove these arguments.");
```

Comment:
this happens when user calls torch.ops.quantized.conv2d directly or in calling quantized functional ops in the serialized code.

Reply:
I don't think this can be runnable, don't we need to modify the type as we do in `qconvAttributeAssignmentSpecialHandlingHack` in order to make it work?